blacksauce/src/menu.html

```diff
@@ -0,0 +1,100 @@
+---
+layout: layouts/default.njk
+title: Sample Menu
+tags: nav
+---
+
+<div class="background--imgMenu">
+    <div class="u-transparentBlack">
+        <h1>Sample Menu</h1>
+        <p class="u-white"> The events we cater vary in size from a couple dozen to a couple hundred people.
+            They vary
+            in theme from
+            informal
+            family celebration to formal seated dinner. We don’t operate off of stock menus because most of the menus
+            we
+            generate are seasonal, specific to the context of event, and developed in close consultation with our
+            clients.
+            Above are some examples of menus we’ve done in recent months, but again, they are event specific. The best
+            way
+            – in
+            fact, the only way – to know whether or not we are suited to do an event is to contact us. We would love to
+            hear
+            from you.</p>
+    </div>
+</div>
+
+<div class="grid--menuPage">
+
+    <p class="p--menuType p--formalDinners">Formal Dinners</p>
+    <ul class="ul--formalDinners">
+        <li class=" p--menuSubhead">-Hors d’oeuvres-</li>
+
+        <li>pork belly w. smoked rhubarb jam</li>
```

Comment:
It would be more semantic to structure this with [submenus like so](https://codepen.io/angeliquejw/pen/yGbBKz?editors=1000).